src/events/ChannelCreateEvent.ts

```diff
@@ -1,23 +1,18 @@
 import BaseEvent from "../structures/BaseEvent";
 import DiscordClient from "../client/client";
-import { GuildChannel, TextChannel } from "discord.js";
+import { GuildChannel, MessageEmbed, TextChannel } from "discord.js";
+import { getAudituser, ModLog } from "../utils/functions/mod";
 
 export default class Event extends BaseEvent {
     constructor() {
         super("channelCreate");
     }
     async run(client: DiscordClient,channel:GuildChannel) {
-        
-        var guildOption = client.guildSettings.get(channel.guildId)?.moderationSettings
-        // make message here as embed
-
-        //make this a function
-        if(!guildOption?.modLogChannel)return
-        var guild = channel.guild
-        var Modchannel = await guild.channels.fetch(guildOption.modLogChannel) as TextChannel
-        if(!Modchannel)return
-        Modchannel.send("embed here")
-
-        //make this a function
+        var data = await getAudituser(channel)
```

Comment:
kindly use const if it doesn't change for the process else use let 
es6 standard